src/ripple/protocol/STObject.h

```diff
@@ -440,7 +437,6 @@ class STObject
     STPathSet const& getFieldPathSet (SField const& field) const;
```

Comment:
Yep - `getFieldPathSet` is used in a few places, whereas `setFieldPathSet` isn't, so I chose to retain one and ditch the other.